BRLS/src/library/payfine/PayFineUI.java

```diff
@@ -5,117 +5,110 @@
 public class PayFineUI {
 
 
-	private enum uI_sTaTe { INITIALISED, READY, PAYING, COMPLETED, CANCELLED };
+    private enum PayFineUIState {INITIALISED, READY, PAYING, COMPLETED, CANCELLED };
 
-	private pAY_fINE_cONTROL CoNtRoL;
-	private Scanner ScAnNeR;
-	private uI_sTaTe StAtE;
+    private PayFineControl control;
+    private Scanner scanner;
+    private PayFineUIState uiState;
 
 	
-	public PayFineUI(pAY_fINE_cONTROL control) {
-		this.CoNtRoL = control;
-		ScAnNeR = new Scanner(System.in);
-		StAtE = uI_sTaTe.INITIALISED;
-		control.SeT_uI(this);
-	}
+    public PayFineUI(PayFineControl payFineControl) {
+        this.control = payFineControl;
+        scanner = new Scanner(System.in);
+        uiState = PayFineUIState.INITIALISED;
+        payFineControl.setUI(this);
+    }
 	
 	
 
-	public void RuN() {
-		DiSpLaYoUtPuT("Pay Fine Use Case UI\n");
+    public void run() {
+        displayOutput("Pay Fine Use Case UI\n");
 		
-		while (true) {
+        while (true) {
 			
-			switch (StAtE) {
+            switch (uiState) {
 			
-			case READY:
-				String PaT_Str = GeTiNpUt("Swipe patron card (press <enter> to cancel): ");
-				if (PaT_Str.length() == 0) {
-					CoNtRoL.CaNcEl();
-					break;
-				}
-				try {
-					long PAtroN_ID = Long.valueOf(PaT_Str).longValue();
-					CoNtRoL.CaRd_sWiPeD(PAtroN_ID);
-				}
-				catch (NumberFormatException e) {
-					DiSpLaYoUtPuT("Invalid patronID");
-				}
-				break;
+            case READY:
```

Comment:
Defect 2 PR#2.
Indentation needs to be addressed for the nested switch statements as per rule 9 of the coding style guideline.

Reply:
fixed, please review again

Reply:
All good.